packages/ember-htmlbars/lib/component.js

```diff
@@ -138,6 +139,21 @@ const Component = View.extend(TargetActionSupport, {
     }
   }),
 
+  templateForName(name, type) {
```

Comment:
@krisselden - I definitely don't think `templateForName` is an API that we need to support.  I'd be fine if this were moved to a simple util that we call...